bin/blazepack.js

```diff
@@ -27,13 +35,13 @@ function validateNewProject(projectName, template) {
     template = 'react';
   }
 
-  if (!availableTemplates.includes(template)) {
-    logError(`Unknown template ${template}, available options are ${availableTemplates.join(',')}`);
+  if (!officialTemplates[template]) {
+    logError(`Unknown template ${template}, available options are ${Object.keys(officialTemplates).join(',')}`);
 
```

Comment:
can we extract `${Object.keys(officialTemplates).join(',')` into a separate variable `availableTemplates` may be?